### PR TITLE
This is my working Skeleton of the Current project

### PR DESCRIPTION
**Pull request by BiggiePete**
This is the new addition of the Inventory page, a working Admin page, and some basic additions to get everything working,

NOTE : Search functionality is currently case sensitive, and non predictive, and also completely un-intuitive, so that is the next major improvement to be made

### CODE REVIEW

File: pacman/pacman/urls.py [hunk 1]
```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
 	path('', include('inventory.urls')),
-    path('admin/', include('admin.urls')),
+    #path('admin/', include('admin.urls')),
```
Comment (xar-tol) [inline pacman/pacman/urls.py:21]: Maybe get rid of this comment since it's not used anymore?

File: pacman/staticfiles/css/inventory.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+@import url(https://fonts.googleapis.com/css?family=Lato:100,300,400,700);
+@import url(https://raw.github.com/FortAwesome/Font-Awesome/master/docs/assets/css/font-awesome.min.css);
```
Comment (xar-tol) [inline pacman/staticfiles/css/inventory.css:2]: Good font package!

File: pacman/staticfiles/css/inventory.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  height: 60px;
+  float: right;
+  padding: 0;
+}
+.top-row{
+  font-size: 16pt;
+  border:1px solid #d9ad77;
+  color:#ddd;
+}
+table,th, td {
```
Comment (xar-tol) [inline pacman/staticfiles/css/inventory.css:34]: Can we make the spacing consistent here?

File: pacman/staticfiles/html/inventory.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	<meta charset="UTF-8" />
+	<link rel="stylesheet" type="text/css" href="{% static 'css/inventory.css'%}" />
+	<Title>Pac-Man</Title>
+	<script src="https://kit.fontawesome.com/afd6aa68df.js" crossorigin="anonymous"></script>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Noto+Sans+Mono:wght@200&display=swap" rel="stylesheet" />
+</head>
+<nav class="search-displace">
+	<h1 class="title">Inventory @ Robot-club</h1>
```
Comment (xar-tol) [inline pacman/staticfiles/html/inventory.html:15]: I assume this is a placeholder, but "Robot-club" should probably be "Robotics Club"

File: pacman/inventory/models.py [hunk 5]
```diff
@@ -1,3 +1,9 @@
 from django.db import models
-
+from django.db.models.base import Model
 # Create your models here.
+class Item(Model):
+    name = models.CharField(max_length=128)
+    quantity = models.IntegerField()
+    description = models.CharField(max_length=256)
```
Comment (xar-tol) [inline pacman/inventory/models.py:7]: Maybe we can add image here at some point and have it be a string that's either a link or file location?

**Review by xar-tol — CHANGES_REQUESTED** [review]
Looks like a good skeleton, but can you remove the BIN files? AKA  "pacman/admin/__pycache__/...", "pacman/inventory/__pycache__/...", etc

File: pacman/pacman/urls.py (hunk 1 above)
Comment (BiggiePete) [reply pacman/pacman/urls.py:21]: For your admin page to work that line would need to be uncommented, as I have it commented to skip by the admin app, and use Django's built-in page for it, not entirely sure why it worked, it just did

**Comment by BiggiePete** [comment]
ill get to removing the caches on monday

**Comment by xar-tol** [comment]
Replaced with #2

**Pull request closed without merging**